pyproject.toml

```diff
@@ -1,3 +1,23 @@
 [build-system]
-requires = ["setuptools", "wheel", "cython"]
+requires = [
+  "setuptools>=18.0",
+  "wheel",
+  "cython",
+  "numpy>=1.23.3",
+  "torch>=1.12",
```

Comment:
According to our documentation we are supporting all PyTorch versions newer than 1.5. Considering v1.5 is very archaic, we can revise that requirement, but I think it would be safer to specify `>=1.10` or `>=1.11`, which are still used pretty widely.